packages/cxl-ui/src/components/cxl-icon-nav-item.js

```diff
@@ -0,0 +1,33 @@
+// Import the LitElement base class and html helper function
+import { LitElement, html } from 'lit-element';
+import cxlThemeStyles from '../styles/cxl-theme-css.js';
+import cxlIconNavItemStyles from '../styles/cxl-icon-nav-item-css.js';
+
+class CXLIconNavItemElement extends LitElement {
+  static get properties() {
+    return {
+      href: {
+        type: String
+      },
+      position: {
+        type: String
+      }
+    };
+  }
+
+  static get styles() {
+    return [cxlThemeStyles, cxlIconNavItemStyles];
+  }
+
+  render() {
+    return html`
+      <li>
```

Comment:
`ul > li` spec doesn't jump across custom element element barriers - `li` is only legit as a direct child.

Reply:
Hmm didn't think of that. Going to try it with vaadin-list-box, same thing with `<cxl-institute-sidebar-item>`

Reply:
I have used to do it in Vue where shadow dom doesn't exist.